src/main/java/controller/RacingGameController.java

```diff
@@ -36,16 +37,34 @@ public List<Car> startRace(MoveStrategy moveStrategy) {
             resultView.printRoundResult(cars);
         }
 
-        resultView.printResult(cars, winningPosition);
         return cars;
     }
 
-    public void updateWinningPosition(int position) {
+    void updateWinningPosition(int position) {
         if (position > winningPosition) {
             winningPosition = position;
         }
     }
 
+    public List<String> findWinningCars() {
+        List<String> winningCars = new ArrayList<>();
+        for (Car car : cars) {
+            addWinningCar(winningCars, car);
+//            if (car.getPosition() == winningPosition) {
+//                winningCars.add(car.getName());
+//            }
+        }
+        return winningCars;
+    }
+
+    void addWinningCar(List<String> winningCars, Car car) {
+        if (car.getPosition() == winningPosition) {
```

Comment:
위치 값을 꺼내서(getter) 비교하기 보다 자동차에게 위치 값을 물어보면 어떨까요?
```suggestion
        if (car.samePosition(winningPosition)) {
```